beta/examples/tensorflow/common/object_detection/datasets/builder.py

```diff
@@ -49,7 +49,7 @@ def num_classes(self):
             return self._dataset_loader.num_classes
         return None
 
-    def _tfds_decoder(self, features_dict):
+    def _tfds_decoder(self, features_dict, include_mask=False):
```

Comment:
The changing the function signature does not look like the pythonic way. Could you use the partial function for decoder_fn. For example:

```
        if self._dataset_type == 'tfrecords':
            decoder_fn = partial(self._dataset_loader.decoder, include_mask=self._include_mask)
        else:
            decoder_fn = self._tfds_decoder
        preprocess_input_fn = get_preprocess_input_fn(self._config, self._is_train)
        preprocess_pipeline = lambda record: preprocess_input_fn(decoder_fn(record))
```

Reply:
Thanks! It looks better